vslib/inc/TrafficForwarder.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "swss/sal.h"
+
+#include <stddef.h>
+#include <sys/socket.h>
+
+static constexpr size_t ETH_FRAME_BUFFER_SIZE = 0x4000;
+static constexpr size_t CONTROL_MESSAGE_BUFFER_SIZE = 0x1000;
+
+namespace saivs
+{
+    class TrafficForwarder
+    {
+    public:
+        virtual ~TrafficForwarder() = default;
+
+    protected:
+        TrafficForwarder() = default;
+
+        static void addVlanTag(
+            _Inout_ unsigned char *buffer,
+            _Inout_ size_t &length,
+            _Inout_ struct msghdr &msg);
+
+        bool sendTo(
```

Comment:
this method also can be static since there are no members, sorry i didnt mentioned to make all methods static

Reply:
Yes, I thought this design you mentioned.
But I hope `TrafficForwarder` could be an abstract interface which means you can did some customized `sendTo` flow by overriding it. How about to make it becomes a virtual function?

Reply:
ok, make this virtual

Reply:
Done